releng/org.eclipse.cdt.target/cdt.target

```diff
@@ -202,6 +192,13 @@
 			     See https://github.com/eclipse-pde/eclipse.pde/issues/675
 			     -->
 			<unit id="com.google.guava.failureaccess" version="0.0.0" />
+
+			<unit id="org.junit" version="4.13.2.v20211018-1956" />
+		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://download.eclipse.org/tools/orbit/simrel/maven-bnd/nightly/latest"/>
+			<!-- may be a temporary problem but https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/nightly is missing the update version of this bundle -->
```

Comment:
I think once https://github.com/eclipse-orbit/orbit-simrel/pull/3 is resolved then the out of sync of the two orbit repos will be resolved.